windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -17,6 +17,18 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
 
 You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
 
+[id="wmco-2-0-2"]
+== Release notes for Red Hat Windows Machine Config Operator 2.0.2
+
+Issued:2021-07-XX
+
+The WMCO 2.0.2 is now available with bug fixes. The components of the WMCO were released in link:https://access.redhat.com/errata/RHSA-XXXX:YYYY[RHSA-XXXX:YYYY].
+
+[id="wmco-2-0-2-bug-fixes"]
+=== Bug fixes
+
+* Previously, when you created a Windows pod, the status would remain stuck in "ContainerCreating." This bug was due to the `BoundServiceAccountTokenVolume` option being enabled by default with {product-title} 4.8. This option caused projected volumes being attached to all of the pods. Finally, the `RunAsUser` option was added to the `SecurityContext` when executing the `os.Chown()` method, resulting in failure. Now, after upgrading to WMCO 2.0.2 prior to upgrading to {product-title} 4.8, he status updates to "Running" once the Windows pod is active. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1975553[**BZ#1975553**])
```

Comment:
We should not mention `Previously, when you created a Windows pod, the status would remain stuck in "ContainerCreating."` as this cannot happen in the context of OCP 4.7 / WMCO 2.0. 
How about something like:
> `BoundServiceAccountTokenVolume` option is being enabled by default in {product-title} 4.8. This option causes projected volumes being attached to all of the pods. In addition, the `RunAsUser` option is added to the `SecurityContext` and  this combinations results in Windows Pods being stuck in `ContainerCreating`. To workaround this customers should upgrade to WMCO 2.0.2 before upgrading their cluster to 4.8.` 

We should also call out https://github.com/kubernetes/kubernetes/issues/102849 as an existing issue for Windows Pods. We should mention that we have a workaround that allows the Pods to go to running but the file permissions are not applied correctly. This applies to WMCO 1.0 also.

Reply:
Added a separate note to the Known Issues to cover this bug. Then I revised the bug fix section a little bit. PTAL